helloworld-rs/src/main/java/org/jboss/as/quickstarts/helloworldrs/HelloWorld.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.jboss.as.quickstarts.rshelloworld;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+/**
+ * A simple REST service which is able to say "Hello World!"
+ *
+ * @author Ashwin Mehendale
+ * @author emartins
+ */
+
+@Path("/")
+public class HelloWorld {
+
+    @GET
+    @Path("/HelloWorld")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String getHelloWorldJSON() {
+        return "<h1>Hello World!</h1>";
```

Comment:
The HTML tags should be removed for plain text probably.

Reply:
done